docs/docker-shim.md

```diff
@@ -161,6 +161,6 @@ You should now see the node Ready for scheduling with the docker Runtime:
 sudo k0s kubectl get nodes -o wide
 
 NAME                                        STATUS   ROLES           AGE    VERSION       INTERNAL-IP   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION    CONTAINER-RUNTIME
-ip-10-0-49-188.eu-west-1.compute.internal   Ready    control-plane   119m   v1.26.1+k0s   10.0.49.188   <none>        Ubuntu 20.04.4 LTS   5.13.0-1022-aws   docker://20.10.16
-ip-10-0-62-250.eu-west-1.compute.internal   Ready    <none>          66m    v1.26.1+k0s   10.0.62.250   <none>        Ubuntu 20.04.4 LTS   5.13.0-1017-aws   docker://20.10.16
+ip-10-0-49-188.eu-west-1.compute.internal   Ready    control-plane   119m   v{{{ extra.k8s_version }}}+k0s   10.0.49.188   <none>        Ubuntu 20.04.4 LTS   5.13.0-1022-aws   docker://20.10.16
```

Comment:
This will break the layout if we reach point releases >9.

Reply:
not sure I follow why.

Reply:
The lines will be misaligned with their headers. Not a big deal, but it will probably look a little bit wonky in the docs and those console snippets will look like "edited after the fact".

Reply:
Yeah, in fact, the raw markdown files look strange now, with placeholder instead of the actual version, not sure how bad is it (e.g. how many people read website and how many read markdown directly).